bin/ch-test

```diff
@@ -35,6 +35,7 @@ Additional options:
   --img-dir DIR          unpacked images directory; same as \$CH_TEST_IMGDIR
   --pack-dir DIR         packed imaged directory; same as \$CH_TEST_TARDIR
   --perm-dir DIR         permissions fixture dir, can repeat; \$CH_TEST_PERMDIRS
+  --fmt FMT              flattened image format (squashfs, tar)
```

Comment:
1. Call it `--pack-format` or `--pack-fmt`?
2. Refer to it as "packed image" rather than "flattened image" (flattened is the directory).